thrift/jaeger.thrift

```diff
@@ -58,7 +58,7 @@ struct Span {
   4:  required i64           parentSpanId # since nearly all spans will have parents spans, CHILD_OF refs do not have to be explicit
   5:  required string        operationName
   6:  optional list<SpanRef> references   # causal references to other spans
-  7:  required i32           flags        # tbd
+  7:  required i32           flags        # a bit field used to propagate sampling decisions. 1 signifies a SAMPLED span, 2 signifies a DEBUG span.
```

Comment:
I was thinking that the field remain `i32` but we assign enums to it. I was assuming that thrift enums are `i32` under the hood, but I'll verify.